modules/tasks.js

```diff
@@ -0,0 +1,164 @@
+const tasks = JSON.parse(localStorage.getItem('tasks')) || [];
+
+function saveTasks() {
```

Comment:
Hey Victor, great job you have done with your project.  However I would like to make some suggestions about ES6 best practices in your code, in that way you can have a more professional and readable code.  In this function you can implement arrow function...with this implementation you can use 1 code line instead of 3 improving the efficiency of your code.